boosters/node-beginner/migrations/20200113152137-repo-association.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
```

Comment:
Just wondering why you put this into its own file as opposed to in the `create-repo` file?

Reply:
Including it as part of the create-repos migration would be valid as well, you can try to refactor it if you want to try it out.

Separating it out to a separate migration allows us to modify the relationship/association in the future without taking down the entire table.